lib/fluent/plugin/in_kafka_group.rb

```diff
@@ -248,14 +252,18 @@ def run
     while @consumer
       begin
         @consumer.each_batch(@fetch_opts) { |batch|
-          es = Fluent::MultiEventStream.new
```

Comment:
For the performance, separating method is better for `in_kafka_group`.

```
@consumer.each_batch(@fetch_opts) { |batch|
  if @tag_source == :record
    process_batch_with_record_tag(batch) # use this patch implementation
  else
    process_batch(batch)  # use existing implementation
  end
}
```

Reply:
Thanks for the suggestion, implemented it. Care to review again? :)